modules/@apostrophecms/ui/ui/apos/components/AposContextMenu.vue

```diff
@@ -116,6 +116,9 @@ const dropdown = ref();
 const dropdownContent = ref();
 const dropdownContentStyle = ref({});
 const arrowEl = ref();
+const menuOffset = ref(
+  Array.isArray(props.menuOffset) ? props.menuOffset : [ props.menuOffset, 0 ]
```

Comment:
This doesn't seem like enough. If they pass `[10]` then their element goes all the way to the left edge of the screen.

If they pass `[10]` the resulting offset should become `[10, 0]`

Reply:
This case straight up breaks the UI but is a valid value, so it needs to be handled

Reply:
Done